contribute.md

```diff
@@ -2,4 +2,4 @@
 
 To contribute a new model, create a [pull request](https://github.com/onnx/models/pull/new/). A pre-defined pull request [template](PULL_REQUEST_TEMPLATE.md) is loaded when creating a new pull request, which describes all the artifacts required for making the contribution.
 
-View an [example submission](./models/image_classification/resnet/README.md) to get a sense of the final output.
+View an [example submission](./models/vision/classification/resnet/README.md) to get a sense of the final output.
```

Comment:
the link is still incorrect. it should be (vision/classification/resnet/README.md)

Reply:
Thanks @prasanthpul.